src/components/DryRunnerExplanation/index.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import TogglableContent from '../TogglableContent'
+import WikiLink from '../WikiLink'
+import { PROBABILITIES } from '../../constants/dryRunner'
+import './index.css'
+
+const CARD_MECHANICS = {
+  PLAYING_CYCLING_DISCARDING: {
+    label: 'Playing, Cycling and Discarding',
+    cards: ['N12', 'N14', 'N22', 'N33'],
+  },
+  CARDS_FROM_DECK_HAND: {
+    label: 'Playing cards from your deck/hand',
+    cards: ['N48', 'S21'],
+  },
+  FREEZE: { label: 'Freeze mechanics', cards: ['W1', 'W8'] },
+  MANA: {
+    label: 'Gaining mana',
+    cards: ['W12', 'W19', 'W10', 'W9', 'W16', 'W8'],
+  },
+  RNG: { label: 'RNG-dependent', cards: ['S3', 'W9', 'W16', 'W8'] },
+  FIRST_TURN: { label: 'First turn restrictions', cards: ['W1', 'F4', 'N63'] },
+  OTHERS: { label: 'Others', cards: ['N8', 'F8'] },
+}
+
+const CARD_ATTRIBUTES = {
+  N12: 'Discards a random non-Pirate card from your hand',
+  N14: 'Will draw one or two cards, depending on the level',
```

Comment:
Is there a reason you use present tense or future?

Reply:
I like being inconsistent 😅